doc/intro.rst

```diff
@@ -18,10 +18,10 @@ and saves it back to disc again::
 
     image = pyvips.Image.new_from_file('some-image.jpg', access='sequential')
     image *= [1, 2, 1]
-    mask = pyvips.Image.new_from_array([[-1, -1, -1],
-                                        [-1, 16, -1],
-                                        [-1, -1, -1]
-                                       ], scale=8)
+    mask = pyvips.Image.new_from_list([[-1, -1, -1],
```

Comment:
Ooof I hadn't though thought of this. I think we'll need to add `[list]` support to `new_from_array` or there will be a of of downstream breakage.

Reply:
https://github.com/libvips/pyvips/blob/df462ca09b8d63c8a5a9cd2f21a6564ddcd3ef08/pyvips/vimage.py#L504

Reply:
Ah of course you're right, I'm not paying attention.